zerver/tests/test_narrow.py

```diff
@@ -460,6 +460,11 @@ def get_query_ids(self) -> Dict[Text, int]:
 
         return query_ids
 
+    def test_use_first_unread_anchor_response(self) -> None:
+        self.login(self.example_email("hamlet"))
+        result = self.get_and_check_messages(dict(use_first_unread_anchor='true'))
+        self.assertIn("anchor", result)
+
```

Comment:
I'd probably just add the assert to an existing test for `use_first_unread_anchor.`